src/styles/_catalog.scss

```diff
@@ -14,9 +14,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
-  }
+    transition: transform 300ms, color 300ms;
+
+    &:not(:nth-child(4n)) {
+      margin-right: 48px;
+    }
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+    &:hover {
```

Comment:
You can add "cursor: pointer", it looks nice when you hover over the card more interactive.

Reply:
yes you are right! thank you for suggestion!